<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Converter</class>
 <widget class="QMainWindow" name="Converter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>209</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Converter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>12</y>
      <width>565</width>
      <height>128</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Единицы измерения</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="Measurments">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <item>
            <property name="text">
             <string>Длина</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Валюта</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Площадь</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Объем</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Масса</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Температура</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Скорость</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Время</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="clearleft">
               <property name="text">
                <string>Очистить лево</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="clearright">
               <property name="text">
                <string>Очитстить право</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="clearall">
             <property name="text">
              <string>Очитстить все</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <property name="leftMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="QLineEdit" name="Number1">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="whatsThis">
          <string/>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>Введите число</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="Select1"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>=</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Number2">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="placeholderText">
          <string>Введите число </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="Select2"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>642</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
